Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/DataEditorForm.java

```diff
@@ -787,6 +787,9 @@ void setSelectedMedia(List<Pair<PhysicalDivision, LogicalDivision>> media) {
      */
     public boolean isSelected(PhysicalDivision physicalDivision, LogicalDivision logicalDivision) {
         if (Objects.nonNull(physicalDivision) && Objects.nonNull(logicalDivision)) {
+            if (physicalDivision.hasMediaPartialView()) {
+                return selectedMedia.contains(new ImmutablePair<>(physicalDivision, physicalDivision.getLogicalDivisions().get(0)));
```

Comment:
At the moment, physical divisions with a media partial view are only assigned one logical division. You cannot create a media partial on your own; you need to use the provided function in the user interface. I added a condition to check if there is a logical division. While it ideally should not be necessary, it serves as a safeguard against an IndexOutOfBoundsException in case someone was manipulating the workpiece.